examples/Example1.cpp

```diff
@@ -24,11 +24,11 @@
 //
 //
 //  This piece of code was developed as part of ICE-DIP project at CERN.
-//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's 
+//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's
 //  7th Framework programme Marie Curie Actions under grant PITN-GA-2012-316596".
 //
 
-#include "../UMESimd.h"
```

Comment:
I don't think that the location of the files will prevent people from finding the code they are interested in, but I also encourage you to write proper documentation to the project, to make life easier to the users. If your library is too difficult to use, no one will use it.